configfiles/fit/config_Lc_Fit_pp13TeV_highSignif.yml

```diff
@@ -0,0 +1,20 @@
+pp13TeVFD:
```

Comment:
To reduce a bit the number of config files, you could have a single config file for the fits with the two entries `pp13TeVFD` and `pp13TeVPrompt`

Reply:
Thanks! I didn't think of this possibility